src/dsl.jl

```diff
@@ -1,64 +1,4 @@
-a `ReactionNetwork` that can be used as input to generation of ODE, SDE, and
-Jump problems.
-
-Most arrows accepted (both right, left, and bi-drectional arrows). Note that
-while --> is a correct arrow, neither <-- nor <--> works. Using non-filled
-arrows (⇐, ⟽, ⇒, ⟾, ⇔, ⟺) will disable mass kinetics and let you cutomize
-reaction rates yourself. Use 0 or ∅ for degradation/creation to/from nothing.
-
-Example systems:
-
-    ### Basic Usage ###
-    rn = @reaction_network begin           # Creates a ReactionSystem.
-        2.0, X + Y --> XY                  # This will have reaction rate corresponding to 2.0*[X][Y]
-        2.0, XY ← X + Y                    # Identical to 2.0, X + Y --> XY
-    end
-
-    ### Manipulating Reaction Rates ###
-    rn = @reaction_network begin
-        2.0, X + Y ⟾ XY                   # Ignores mass kinetics. This will have reaction rate corresponding to 2.0.
-        2.0X, X + Y --> XY                 # Reaction rate needs not be constant. This will have reaction rate corresponding to 2.0*[X]*[X]*[Y].
-        XY+log(X)^2, X + Y --> XY          # Reaction rate accepts quite complicated expressions.
-        hill(XY,2,2,2), X + Y --> XY       # Reaction inis activated by XY according to a hill function. hill(x,v,K,N).
-        mm(XY,2,2), X + Y --> XY           # Reaction inis activated by XY according to a michaelis menten function. mm(x,v,K).
-    end
-
-    ### Multiple Reactions on a Single Line ###
-    rn = @reaction_network begin
-        (2.0,1.0), X + Y ↔ XY              # Identical to reactions (2.0, X + Y --> XY) and (1.0, XY --> X + Y).
-        2.0, (X,Y) --> 0                   # This corresponds to both X and Y degrading at rate 2.0.
-        (2.0, 1.0), (X,Y) --> 0            # This corresponds to X and Y degrading at rates 2.0 and 1.0, respectively.
-        2.0, (X1,Y1) --> (X2,Y2)           # X1 and Y1 becomes X2 and Y2, respectively, at rate 2.0.
-    end
-
-    ### Adding Parameters ###
-    kB = 2.0; kD = 1.0
-    p = [kB, kD]
-    p = []
-    rn = @reaction_network begin
-        (kB, kD), X + Y ↔ XY               # Lets you define parameters outside on network. Parameters can be changed without recalling the network.
-    end
-
-    ### Defining New Functions ###
-    my_hill_repression(x, v, k, n) = v*k^n/(k^n+x^n)
-
-    # may be necessary to
-    # @register_symbolic my_hill_repression(x, v, k, n)
-    # see https://docs.sciml.ai/ModelingToolkit/stable/basics/Validation/#User-Defined-Registered-Functions-and-Types
-
-    r = @reaction_network MyReactionType begin
-        my_hill_repression(x, v_x, k_x, n_x), 0 --> x
-    end
-
-    ### Simulating Reaction Networks ###
-    probODE = ODEProblem(rn, args...; kwargs...)        # Using multiple dispatch the reaction network can be used as input to create ODE, SDE and Jump problems.
-    probSDE = SDEProblem(rn, args...; kwargs...)
-    probJump = JumpProblem(prob,aggregator::Direct,rn)
-"""
```

Comment:
The first documentation of `@reactio_network`, but now not even a docstring and just a weird remnant. remove.